acl.n3

```diff
@@ -0,0 +1,182 @@
+
+agent   a rdf:Property;
+        rdfs:label "agent";
+        rdfs:comment "A person or social entity to being given the right";
+        rdfs:domain     Authorization;
+        rdfs:range      foaf:Agent.
+
+origin   a rdf:Property;
+        rdfs:label "origin";
+        rdfs:comment """A web application, identified by its Origin, such as
+        <https://scripts.example.com>, being given the right.
+        When a user of the web application at a certain origin accesses the server,
+        then the browser sets the Origin: header to warn that a possibly untrusted webapp
+        is being used.
+        Then, BOTH the user AND the origin must have the required access.""";
+        rdfs:seeAlso <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin>;
+        rdfs:domain     Authorization;
+        rdfs:range      Origin.
+
+agentClass   a rdf:Property;
+        rdfs:label "agent class";
+        rdfs:comment "A class of persons or social entities to being given the right";
+        rdfs:domain     Authorization;
+        rdfs:range      rdfs:Class.  # Typically be subclass of foaf:Agent.
+                        # For public access, use foaf:Agent.
+
+AuthenticatedAgent a rdfs:Class;
+        rdfs:subClassOf foaf:Agent;
+        rdfs:label "Anyone authenticated";
+        rdfs:comment """A class of agents who have been authenticated.
+In other words, anyone can access this resource, but not anonymously.
+The social expectation is that the authentication process will provide an
+identify and a name, or pseudonym.
+(A new ID should not be minted for every access: the intent is that the user
+is able to continue to use the ID for continues interactions with peers,
+and for example to develop a reputation)
+""" .
+
+agentGroup a rdf:Property;
+          rdfs:label "agent group";
+          rdfs:comment """A group of persons or social entities to being given the right.
+          The right is given to any entity which is a vcard:member of the group,
+          as defined by the document received when the Group is dereferenced.""";
+          rdfs:domain     Authorization;
+          rdfs:range      vcard:Group.  # Must be subclass of foaf:Agent.
+                          # For public access, use foaf:Agent.
+
+accessTo
+    a rdf:Property;
+        rdfs:label "to";
+        rdfs:comment "The information resource to which access is being granted.";
+        rdfs:domain     Authorization;
+        rdfs:range      gen:InformationResource.
+
+accessToClass
+    a rdf:Property;
+        rdfs:label "to all in";
+        rdfs:comment "A class of information resources to which access is being granted.";
+        rdfs:domain     Authorization;
+        rdfs:range      rdfs:Class.
+
+default
+    a rdf:Property;
+        rdfs:label "default access for things in this";
+        rdfs:comment """If a resource has no ACL file (it is 404),
+        then access to the resource if given by the ACL of the immediately
```

Comment:
```suggestion
        then access to the resource is given by the ACL of the immediately
```

Reply:
@TallTed Thanks. There are updates indeed in the pipeline. I just wanted to get the current version in as a standalone PR. If you don't mind, we can follow-up with updates in a different PR. I'll include your suggestion there.